packages/nextjs-kit/src/components/recipe.tsx

```diff
@@ -0,0 +1,90 @@
+import React from 'react';
+import Image from 'next/image';
+import type { ImageProps } from 'next/image';
+import { useRouter } from 'next/router';
+
+interface RecipeProps {
+	title: string;
+	category: string;
+	imageProps?: ImageProps;
+	ingredients: string[];
+	instructions: string;
+	children?: JSX.Element;
+}
+
+/**
+ *
+ * @param props - The props needed for the Recipe component
+ * @param props.title - The title of the recipe
+ * @param props.category - The identifying category of your recipe
+ * @param props.imageProps - All props the user wishes to pass to the next/image component @see {@link https://nextjs.org/docs/api-reference/next/image} for all next/image documentation
+ * @param props.ingredients - An array of strings with ingredients to be displayed
+ * @param props.instructions - Instructions from a CMS, usually a string of raw HTML. This string will be set as `dangerouslySetInnerHTML`
+ * @returns A recipe component with content and an optional image passed by the user
+ * @remarks
+ * imageProps is an optional prop to be used if there is an image to be associated with the content.
+ * If imageProps is used it is required that the user passes in value for the images src
+ */
+const Recipe: React.FC<RecipeProps> = ({
```

Comment:
I thought this was going to get some more refactoring instead of just a copy -> paste from the original component?

Reply:
Ah, I had thought that the refactoring to do here was only to add support for different recipe fields which we decided to use `children` for. Aside from the new comments, what additional areas are you thinking should be looked at for refactoring?

Reply:
Oh yes, that sounds familiar, my bad! 😅